test/es-module/test-esm-loader-http-imports.mjs

```diff
@@ -42,12 +47,29 @@ const {
 } = server.address();
 
 /**
- * ! If more cases are added to this test, they cannot (yet) be concurrent because there is no
- * ! `afterAll` teardown in which to close the server.
+ * ! TODO(MoLow): can change to concurrency:true when `after` hook is added
+ * ! and server.close() moves to `after`
```

Comment:
😍

Reply:
This can be addressed already, now that the `after` PR has landed.